src/create_disk.sh

```diff
@@ -103,7 +103,10 @@ set -x
 # Partition and create fs's. The 0...4...a...1 uuid is a sentinal used by coreos-gpt-setup
 # in ignition-dracut. It signals that the disk needs to have it's uuid randomized and the
 # backup header moved to the end of the disk.
-# Pin /boot and / to the partition number 3 and 4 respectively
+# Pin /boot and / to the partition number 3 and 4 respectively. Also insert reserved
+# partitions on aarch64/ppc64le to keep the 1,2,3,4 partition numbers aligned across
+# x86_64/aarch64/ppc64le. s390x is quite special so we decided not to try to achieve
+# partition parity there. https://github.com/coreos/fedora-coreos-tracker/issues/855
```

Comment:
More to the point, a bare metal install onto an s390x DASD translates the GPT to DASD partitions, and we only get three of those.

Reply:
That's good context. Will update to be more explicit.